ast/dml_test.go

```diff
@@ -264,7 +267,7 @@ func (tc *testDMLSuite) TestDeleteTableIndexHintRestore(c *C) {
 		return node.(*DeleteStmt)
 	}
 	RunNodeRestoreTest(c, testCases, "%s", extractNodeFunc)
-}
+}*/
```

Comment:
Are you really sure that we do not retain these functions at this stage?

Reply:
These are definitely mysql feature even tidb utilize these options pretty much, we still need to think about a pure/almost pg compatible parser. BTW a lot of conflicts are being expected, let's see whether it is a good idea then.

Reply:
Instead of merging directly into the main branch, I suggest that we divide a branch to complete this series of functions and solve the expected conflicts.

Reply:
It is a good idea, please check the new PR #24